node/utils/ioMessage.ts

```diff
@@ -36,6 +36,24 @@ export const toClusterIOMessage = (segment: Segment, content: string, id: string
   `ProductCluster-id.${id}::${content}`
 )
 
+export const toSKUIOMessage = (field: string) => (segment: Segment, content: string, id: string) => toIOMessage(
```

Comment:
@tlgimenes I didn't get why we use the closure here.
I didn't find any case where we called the nested functions in different places.
Removing the indirection (e.g. `toSKUIOMessage = (field: string, segment: Segment, content: string, id: string)`) makes sense here?